lib/quick_travel/adapter.rb

```diff
@@ -105,6 +84,14 @@ def to_hash
       instance_values
     end
 
+    def to_s
+      if defined? @to_s
```

Comment:
I can't see anywhere that `@to_s` gets defined, should this method just be
```
def to_s
  @to_s ||= super
end
```

Reply:
in which case you can just delete the method right?

Reply:
This was added because serialization helpers in downstream projects expected the method to be defined, we used the defined condition as a way to avoid breaking any modifications/monkey patches others may be using in downstream projects

Reply:
Yeah only difference between the code as written and the suggested change @HellRok is that `super` would be called if `@to_s` was falsey. This would be undesirable. `@to_s` is the value as provided by QT and should be respected no matter the value.